kuma/wiki/templates/wiki/revision.html

```diff
@@ -8,7 +8,7 @@
 {% block content %}
 
   <article>
-    <h1>{{ document.title }}</h1>
+    <h1><a href="{{ url('wiki.document', document.full_path, locale=document.locale) }}">{{ _('{title}')|fe(title=document.title) }}</a></h1>
```

Comment:
I always get this stuff confused. Bottom line is that `document.title` comes from users, so we don't trust it. We store any markup un-escaped, so it needs to be escaped in the template.

I.e., @SebastianZ - please revert the element content back to `{{ document.title }}`